source/_integrations/nanoleaf.markdown

```diff
@@ -50,3 +60,46 @@ name:
 4. The output should include the auth token like *{"auth_token":"xxxxxxxxxxxxxxxxxxxxx"}*, copy the resulting token into your configuration
 
 If you get a 403 Forbidden message, you probably did not press the *ON* button long enough. The time-frame to get a valid token is only 30 seconds, so you have to be quick to issue the curl request.
+
+### Using the Nanoleaf Device
+
+The nanoleaf device is a standard light, so can be used with standard Home Assistant Light services for automations and scripts. For full details see the [Light Integrations Page.](/integrations/light/)
+
+### Getting Useful Values for Automations and Scripts
+
+These examples were written for version 2021.2 with a Nanoleaf Shapes device on version 5.0.0 of firmware, so newer versions may have introduced or changed functionality.
+
+First set up some effects with the Nanoleaf app, and have them installed on the device.
+
+Once installed, go to "Developer Tools" and find the state of the device, e.g., `light.bedroom_triangles`. This will help you discover the names of the effects that you can use with the device. You should end up with something like this:
+
+```yaml
+# Example nanoleaf state:
+# Note that this is all device specific, you will need to look up your own device's state
+min_mireds: 154
+max_mireds: 833
+effect_list:
+  # These effects are device specific, and have been installed through the nanoleaf app
+  - Beatdrop
+  - Sundown
+  - Waterfall
+  - Vibrant Sunrise
+brightness: 155
+effect: Bedtime
+friendly_name: Triangles
+icon: 'mdi:triangle-outline'
+supported_features: 55
+```
+
+Now you can call the `light.turn_on` service in a script for example:
+```yaml
+# Example alarm script
+alias: Alarm Sequence
+sequence:
+  - service: light.turn_on
+    data:
+      # Note - this is one of the effects we discovered in the state
+      effect: Vibrant Sunrise
+    entity_id: light.bedroom_triangles
+mode: single
+```
```

Comment:
This example should not be in this part of the documentation, as it is not specific or unique for Nanoleaf.

Reply:
Ok, dropped all the examples.